osquery/tables/system/posix/yum_sources.cpp

```diff
@@ -0,0 +1,107 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+
+namespace osquery {
+namespace tables {
+
+const std::string kYumConf { "/etc/yum.conf" };
+const std::string kYumReposDir { "/etc/yum.repos.d" };
+
+void parseYumConf(const std::string& source, QueryData& results, std::string& repos_dir) {
+  // Default value
+  repos_dir = kYumReposDir;
+
+  std::string content;
+  if (!readFile(source, content)) {
+    return;
+  }
+
+  std::string section = "";
+  for (const auto& line : osquery::split(content, "\n")) {
+    auto size = line.size();
```

Comment:
Probably should left/right trim whitespaces before checking lines!

Reply:
split() already trims: https://github.com/facebook/osquery/blob/master/osquery/core/conversions.cpp#L377

Reply:
:+1: Never mind.